make/release.mk

```diff
@@ -5,30 +5,31 @@
 ## @category Release
 CMREL_KEY ?=
 
-.PHONY: release
+.PHONY: release-artifacts
 # Build all release artifacts which might be run or used locally, except
 # for anything signed.
-release-artifacts: server-binaries cmctl kubectl-cert_manager helm-chart static-manifests all-containers
+release-artifacts: server-binaries cmctl kubectl-cert_manager helm-chart release-container-bundles release-manifests
 
-.PHONY: release-signed
+.PHONY: release-artifacts-signed
 # Same as `release`, except it also signs the Helm chart. Requires CMREL_KEY
 # to be configured.
-release-artifacts-signed: release-artifacts helm-chart-signature
+release-artifacts-signed: release-artifacts
+	$(MAKE) --no-print-directory helm-chart-signature
```

Comment:
note: I did this for a good reason in #4554 but I foolishly didn't document why :grin: The end result is the same so I've changed it to how it looked then, just so whatever the reason was doesn't crop up again!